projects/docs/datasets_overview.md

```diff
@@ -5,12 +5,12 @@ You have been matched with 4-6 students from your pod to form a project group. B
 In this section, you will find materials to help you get started on your project. These include a brief background, an introductory video (except _Heatwaves_ and _ENSO impact on precipitation and temperature_), a project map, a data exploration notebook, as well as some further reading recommendations.
 
 You can find project materials in the left side panel or by clicking on the project titles below.
-- [Sea level rise](./project-notebooks/Sea_level_rise.ipynb)
-- [Regional precipitation variability and extreme events](./project-notebooks/Regional_precipitation_variability.ipynb)
-- [Change in Earth’s albedo and its dependence on land cover changes in the past 20 years](./project-notebooks/Surface_albedo_and_land_use.ipynb)
-- [Monitoring and mapping wildfires using satellite data](./project-notebooks/Wildfires_and_burnt_areas.ipynb)
-- [Ocean acidification](./project-notebooks/Ocean_acidification.ipynb)
-- [The impact of ENSO on precipitation and temperature](./project-notebooks/ENSO_impact_on_precipitation_and_temperature.ipynb)
-- [Heatwaves: Assessing the dynamic interactions of the atmosphere and land](./project-notebooks/Heatwaves.ipynb)
+- [Sea level rise](../project-notebooks/Sea_level_rise.ipynb)
+- [Regional precipitation variability and extreme events](../project-notebooks/Regional_precipitation_variability.ipynb)
```

Comment:
@zanemit I think this needs to be from root of the projects so it should be `projects/project-notebooks/...` could you update the others? Is there a reason you believe it should be `../` those were my thoughts of the top of my head


```suggestion
- [Regional precipitation variability and extreme events](projects/project-notebooks/Regional_precipitation_variability.ipynb)
```

Reply:
Hm. I haven't managed to set up a local build, but ctrl+click in VScode opens the correct file with the `..\` syntax. It cannot find the file with the syntax we had (which doesn't work in the built book) or the one you have suggested. The location of this file is `projects/docs`, so I think it makes sense that it needs to be guided one directory up. `projects/project-notebooks/<filename>` would be the syntax if `projects` was a folder inside `docs`, no?

Reply:
Yeah I get it works in VS code ^^ unfortunatly during the book build everything gets sym linked into the book folder and it is built from there so I do think projects is needed, see below:

![image](https://github.com/ClimateMatchAcademy/course-content/assets/18068320/5cfce337-af27-436b-83e6-2941af34dd50)

Reply:
Thanks for explaining! Done now!